.github/workflows/release.sh

```diff
@@ -129,29 +147,37 @@ pick_otp_vsn() {
         fi
     done <<<"${kerl_releases_reversed}"
 
+    # Search for a version to apply the pipeline to
     while read -r release; do
         if [[ ${release} =~ ^[0-9].*$ ]]; then
             local major=${release%%.*}
             if [[ ${oldest_supported} == undefined ]]; then
                 echo "Couldn't determine oldest support version. Exiting..."
                 exit 1
             fi
+
             if [[ ${major} -lt ${oldest_supported} ]]; then
                 continue
             fi
 
             local filename_no_ext
             filename_no_ext=$(filename_no_ext_for "${release}")
+
             pushd "${global_INITIAL_DIR}" >/dev/null || exit 1
-            if test -f _RELEASES && grep "${filename_no_ext} " _RELEASES; then
-                continue
+            if [[ -f _RELEASES ]]; then
+                local found
+                found=$(grep "${filename_no_ext} " _RELEASES)
+                if [[ -n "${found}" ]]; then
+                    continue
+                fi
             fi
```

Comment:
Scope creep is always so tempting 😄 Here, I waste your review time by making for a more "consistent" scripting experience.